app/views/homepage/index.html.erb

```diff
@@ -96,7 +96,7 @@
 
     <%= render "homepage/promotion-slots" %>
 
-    <hr class="home-divider" id="more-on-govuk" aria-hidden="true" />
+    <hr class="home-divider" aria-hidden="true" />
```

Comment:
Could we remove this element and put the border on the element preceding it?

```suggestion
```

Reply:
Yep you are right. I added this early on and see now there's no need for it. Removed.

Reply:
Just one thing to mention. Promotional slots has a `border-top` but the following section (More on GOV.UK*) currently doesn't have one. I think this should probably be added in the PR to update that section? https://github.com/alphagov/frontend/pull/3039

Reply:
I think that where the border is will be sorted out when all of the slices are put together and we can see them all together.

The convention for margin spacing is to put it at the bottom of an element (also helps avoid margin collapsing), so we could do the same with the borders for each section as well.

Reply:
The only issue we might have here is that in the *Government Activity* section - the `border-top`s are above the content / headings in each column which creates a gap. If we have a border-bottom on the proceeding section that might be a bit tricky to replicate?

Reply:
Good point - so this needs to be done differently. Let's tackle these borders when all the slices are together and it'll be easier to see what needs to be done across the whole page layout.